AirCasting/Graph/Graph.swift

```diff
@@ -213,7 +238,61 @@ struct Graph: UIViewRepresentable {
             let chartDataEntry = ChartDataEntry(x: timeInterval, y: measurement.value)
             return chartDataEntry
         }) ?? []
-        uiView.updateWithEntries(entries: entries, isAutozoomEnabled: isAutozoomEnabled)
+        
+        if entries.count > 1000 {
```

Comment:
it would be nice to put this 1000 into a `let` with a descriptive name (what it actually stands for?). This way it will be easier to understand what this line means without reading further down.

Reply:
I'd also extract this whole `if/else` into a separate function and inside I'd invert the condition using guard:
``` Swift
func cleverNameHere(entries: [Entry], ...) {
   guard entries <= 1000 else {
      uiView.updateWithEntries(entries: entries, isAutozoomEnabled: isAutozoomEnabled)
      return
   }
   let simplifiedPoints = SwiftSimplify.simplify(entries, tolerance: 0.000000001, highestQuality: true)
   ...
}
```